lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -400,7 +400,9 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
         }
 
         SmallVector<char, 64> Buf;
-        StringRef Name = (VD->getName().str() + Twine(Kind)).toStringRef(Buf);
+        // TODO: Handle special names
```

Comment:
Subscripts can definitely get here, so we'll need to have some way of handling it.